### ISSUE TITLE
UI Component - Wallet Overview

### ISSUE THREAD

**Issue by cammellos** [status-im/status-react#13554]
This issue is to track the creation of the component: **Wallet Overview**

You can access the link at https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=470%3A23418.

Please have a chat with the design team before starting with the component, and don't forget to add sizing to the issue

Charts are out of scope for now
